client/src/components/HomePage.jsx

```diff
@@ -0,0 +1,23 @@
+import ListLatestEntries from './ListLatestEntries'
+
+export default function HomePage() {
+
+    //fetch async function to db where entries are
+
+  return (
+    <div>
+        <br />
```

Comment:
I'd use `<br />`s in HTML sparingly. Most of the `<br/>`s should be handled with `margin` on your elements. 
This one can be a `margin-top` in the `img` element below. It gives you more flexibility and more control over your design.

Reply:
great insight. thank you for helping me make this connection.